api/api.go

```diff
@@ -34,12 +35,28 @@ func (a *API) Router() *fiber.App {
 	return a.router
 }
 
+func getReadBufferSize() int {
+	bufferSizeStr, exists := os.LookupEnv("GATUS_API_READ_BUFFER_SIZE")
+	if !exists {
+		return 4096 // Default value
+	}
+
+	bufferSize, err := strconv.Atoi(bufferSizeStr)
+	if err != nil {
+		log.Printf("Error converting GATUS_API_READ_BUFFER_SIZE to integer: %s", err.Error())
+		return 4096 // Default value in case of conversion error
+	}
+
```

Comment:
```suggestion
	bufferSize, err := strconv.Atoi(bufferSizeStr)
	if err != nil {
		log.Printf("Error converting GATUS_API_READ_BUFFER_SIZE to integer: %s", err.Error())
		return 4096 // Default value in case of conversion error
	}
```